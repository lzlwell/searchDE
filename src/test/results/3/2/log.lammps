LAMMPS (30 Aug 2012)
units		metal
boundary	p p p
atom_style	atomic
read_data	struct.lammps
  orthogonal box = (0 0 0) to (17.1317 10 50)
  1 by 1 by 1 MPI processor grid
  300 atoms

pair_style      airebo 3.0
pair_coeff      * * CH.airebo C H

mass	1	12.0
mass	2	1.0

group		fixPosH id <> 1 120
120 atoms in group fixPosH
group		fixPosL id <> 121 239
119 atoms in group fixPosL
group		optPos  id <> 240 1000
61 atoms in group optPos
fix             0       fixPosH setforce NULL 0.0 NULL
fix             1       fixPosH aveforce 0.0 0.0 0.0
fix		2	fixPosL setforce 0.0 0.0 0.0
fix 		3	optPos	setforce NULL 0.0 NULL

neighbor        0.2 bin
neigh_modify	every 1 delay 0 check yes

timestep	0.002
thermo 		1
thermo_style    custom step  pe etotal lx ly lz 

dump		1 all xyz 10000000 dump.atom
dump_modify	1 format "%d  %20.15g  %20.15g  %20.15g" 
#fix 		8 all box/relax iso 0.0 
min_style	cg
minimize	1.0e-12 1.0e-12 10 10
Memory usage per processor = 3.26918 Mbytes
Step PotEng TotEng Lx Ly Lz 
       0   -673.89097   -673.89097      17.1317           10           50 
       1   -896.83278   -896.83278      17.1317           10           50 
       2   -1013.0743   -1013.0743      17.1317           10           50 
       3   -1073.0335   -1073.0335      17.1317           10           50 
       4   -1112.8199   -1112.8199      17.1317           10           50 
       5   -1133.1037   -1133.1037      17.1317           10           50 
       6   -1151.7594   -1151.7594      17.1317           10           50 
       7   -1166.1087   -1166.1087      17.1317           10           50 
       8   -1174.2355   -1174.2355      17.1317           10           50 
       9   -1180.2657   -1180.2657      17.1317           10           50 
      10   -1185.5185   -1185.5185      17.1317           10           50 
Loop time of 0.365703 on 1 procs for 10 steps with 300 atoms

Minimization stats:
  Stopping criterion = max force evaluations
  Energy initial, next-to-last, final = 
        -673.890969343     -1180.26572711      -1185.5185166
  Force two-norm initial, final = 1525.94 21.4503
  Force max component initial, final = 689.548 12.4267
  Final line search alpha, max atom move = 0.00754869 0.0938054
  Iterations, force evaluations = 10 10

Pair  time (%) = 0.137998 (37.7351)
Neigh time (%) = 0.224131 (61.2877)
Comm  time (%) = 0.00150037 (0.410269)
Outpt time (%) = 0.000149012 (0.0407466)
Other time (%) = 0.00192428 (0.526185)

Nlocal:    300 ave 300 max 300 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Nghost:    2461 ave 2461 max 2461 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Neighs:    0 ave 0 max 0 min
Histogram: 1 0 0 0 0 0 0 0 0 0
FullNghs:  45610 ave 45610 max 45610 min
Histogram: 1 0 0 0 0 0 0 0 0 0

Total # of neighbors = 45610
Ave neighs/atom = 152.033
Neighbor list builds = 10
Dangerous builds = 10
min_modify line quadratic

clear
