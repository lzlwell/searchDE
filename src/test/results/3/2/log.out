LAMMPS (30 Aug 2012)
Reading data file ...
  orthogonal box = (0 0 0) to (17.1317 10 50)
  1 by 1 by 1 MPI processor grid
  300 atoms
120 atoms in group fixPosH
119 atoms in group fixPosL
61 atoms in group optPos
Setting up minimization ...
Memory usage per processor = 3.26918 Mbytes
Step PotEng TotEng Lx Ly Lz 
       0   -673.89097   -673.89097      17.1317           10           50 
       1   -896.83278   -896.83278      17.1317           10           50 
       2   -1013.0743   -1013.0743      17.1317           10           50 
       3   -1073.0335   -1073.0335      17.1317           10           50 
       4   -1112.8199   -1112.8199      17.1317           10           50 
       5   -1133.1037   -1133.1037      17.1317           10           50 
       6   -1151.7594   -1151.7594      17.1317           10           50 
       7   -1166.1087   -1166.1087      17.1317           10           50 
       8   -1174.2355   -1174.2355      17.1317           10           50 
       9   -1180.2657   -1180.2657      17.1317           10           50 
      10   -1185.5185   -1185.5185      17.1317           10           50 
Loop time of 0.365703 on 1 procs for 10 steps with 300 atoms

Minimization stats:
  Stopping criterion = max force evaluations
  Energy initial, next-to-last, final = 
        -673.890969343     -1180.26572711      -1185.5185166
  Force two-norm initial, final = 1525.94 21.4503
  Force max component initial, final = 689.548 12.4267
  Final line search alpha, max atom move = 0.00754869 0.0938054
  Iterations, force evaluations = 10 10

Pair  time (%) = 0.137998 (37.7351)
Neigh time (%) = 0.224131 (61.2877)
Comm  time (%) = 0.00150037 (0.410269)
Outpt time (%) = 0.000149012 (0.0407466)
Other time (%) = 0.00192428 (0.526185)

Nlocal:    300 ave 300 max 300 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Nghost:    2461 ave 2461 max 2461 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Neighs:    0 ave 0 max 0 min
Histogram: 1 0 0 0 0 0 0 0 0 0
FullNghs:  45610 ave 45610 max 45610 min
Histogram: 1 0 0 0 0 0 0 0 0 0

Total # of neighbors = 45610
Ave neighs/atom = 152.033
Neighbor list builds = 10
Dangerous builds = 10
